src/pipeline_v3/tests/integration/test_e2e_integration.py

```
@@ -127,6 +162,15 @@ async def test_document_ingestion(
                         assert model in markdown, f"Expected model {model} not found"
 
             except Exception as e:
+                print(f"Exception during processing: {type(e).__name__}: {e}")
+                print(f"Result before exception: {result if 'result' in locals() else 'No result yet'}")
+
+                # Check if this is an API key related error
+                error_str = str(e).lower()
+                if any(key_phrase in error_str for key_phrase in ['api key', 'authentication', 'unauthorized', 'forbidden']):
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)